Description: Fix callgrind_control that should look for valgrind.bin and not valgrind
Origin: vendor
Bug-Debian: http://bugs.debian.org/659215
Author: Pierre Habouzit <madcoder@debian.org>
Reviewed-by: Alessandro Ghedini <ghedo@debian.org>
Last-Update: 2012-05-10

--- a/callgrind/callgrind_control.in
+++ b/callgrind/callgrind_control.in
@@ -29,7 +29,7 @@
   @pids = ();
   open LIST, "vgdb $vgdbPrefixOption -l|";
   while(<LIST>) {
-      if (/^use --pid=(\d+) for \S*?valgrind\s+(.*?)\s*$/) {
+      if (/^use --pid=(\d+) for \S*?valgrind\.bin\s+(.*?)\s*$/) {
 	  $pid = $1;
 	  $cmd = $2;
 	  if (!($cmd =~ /--tool=callgrind/)) { next; }
